app/helpers/ember_rails_helper.rb

```diff
@@ -8,6 +8,11 @@ def render_ember_app(name, &block)
 
     head, body = markup_capturer.capture
 
-    render text: EmberCli[name].index_html(head: head, body: body).html_safe
+    template = EmberCli[name].index_html(head: head, body: body).html_safe
+    begin
+      render text: template
+    rescue ArgumentError
+      render body: template
```

Comment:
It looks like [`render` accepts an `:html` option](https://github.com/rails/rails/blob/e1a72fef7de9a5c0871b0ef1a4acb9483d2bf1f3/guides/source/upgrading_ruby_on_rails.md#rendering-content-from-string), should we use that instead?